src/analysis/clustering.jl

```diff
@@ -139,12 +140,51 @@ Alias to cluster_series.
 
 # Returns
 - `Vector` : Cluster ids indicating which cluster each scenario belongs to.
+
+# Example
+One can cluster scenarios based on a single Metric, passing a Matrix of outcomes for each
+timestep and scenario:
+
+    ```julia
+# Matrix of outcomes
+s_tac = ADRIA.metrics.scenario_total_cover(rs)
+
+# Cluster scenarios
+num_cluster = 6
+clusters = ADRIA.analysis.cluster_series(s_tac, num_clusters)
+```
+And perform multiple clusterings, based on multiple Metrics, passing a 3-dimensional Array

```

Comment:
New line between sections/code blocks please

```suggestion

And perform multiple clusterings, based on multiple Metrics, passing a 3-dimensional Array

```